README.rst

```diff
@@ -7,6 +7,8 @@ Async http client/server framework
    :width: 64px
    :alt: aiohttp logo
 
+|
```

Comment:
This is to fix logo being rendered in line with badges: https://pypi.org/project/aiohttp/

Reply:
Ref: https://stackoverflow.com/questions/8218039/how-to-add-extra-whitespace-between-section-header-and-a-paragraph